pykern/pkcli/fmt.py

```diff
@@ -17,13 +18,19 @@ def run(path):
     _black(path)
 
 
-def diff(path):
+def diff(path_or_paths):
     """Run diff on file comparing formated vs. current file state
 
     Args:
         path (object): string or py.path to file or directory
     """
-    _black(path, "--diff", "--check", "--no-color")
+    for p in (
+        (path_or_paths,)
+        if isinstance(path_or_paths, py._path.local.LocalPath)
```

Comment:
Don't reach inside`py`. `_path` is private. The way to do this is [done in pkconst](https://github.com/radiasoft/pykern/blob/10d95c24369e847fc12085bdce9239fe7d12e3c4/pykern/pkconst.py#L26) and exported. However, it's better to change the definition of `fmt` (above).

Reply:
81558f6e653269c07a7286f77898de84b03955fc